config/install/field.field.node.starships.field_passen.yml

```diff
@@ -0,0 +1,22 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_passen
+    - node.type.starships
+id: node.starships.field_passen
+field_name: field_passen
+entity_type: node
+bundle: starships
+label: Passengers
+description: 'The number of non-essential people this starship can transport'
```

Comment:
Add a period at the end of the help text for consistency with other help text.
![starships_field_passengers_help_text](https://github.com/notS0Solo/swapimigration/assets/1434979/e4ee52f3-7574-4750-9ca5-6b004ef99ae3)